src/main/java/com/saralein/server/Controller/FileController.java

```diff
@@ -0,0 +1,45 @@
+package com.saralein.server.Controller;
+
+import com.saralein.server.request.Request;
+import com.saralein.server.response.FileHelper;
+import com.saralein.server.response.ResponseBuilder;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+public class FileController implements Controller {
+    private final Request request;
+    private final File resource;
+    private final FileHelper fileHelper;
+
+    public FileController(Request request, File resource, FileHelper fileHelper) {
+        this.request = request;
+        this.resource = resource;
+        this.fileHelper = fileHelper;
+    }
+
+    public byte[] createResponse() {
```

Comment:
This is related to a previous comment.

Do you think that a `Controller` should be responsible for converting a response to bytes?